<html lang="en"><!-- Beautiful Jekyll | MIT license | Copyright Dean Attali 2016 --><head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, viewport-fit=cover">
  
  <title>Avi Karn, Ph.D.</title>

  <meta name="author" content="Â© Avinash Karn">
  <meta name="theme-color" content="#a50509">
  <meta name="description" content="Hello, my name is Avinash " avi"="" karn.="" i="" hold="" a="" ph.d.="" degree="" in="" plant="" sciences="" from="" the="" university="" of="" missouri-columbia,="" and="" bachelor="" science="" agricultural="" truman="" state="" kirksville,="" missouri.="" specialize="" breeding,="" statistical="" genetics="" computational="" biology,="" possess="" an="" excellent="" knowledge="" experience="" conducting="" big="" analysis="" on="" genotypic="" phenotypic="" data="" sets.="" intend="" to="" use="" my="" education="" expertise="" pursuing="" career="" research="" organization="" that="" is="" involved="" improve="" human="" nutrition,="" health="" environment="" for="" both="" developed="" developing="" communities."="">
  <meta name="keywords" content="Avinash Karn PhD" ,="" "avinash="" karn",="" "avi="" "quantitative="" genetics",="" "cornell",="" "food="" blog",="" "ampseq",="" "computational="" biology",="" "nir",="" "qtl="" mapping",="" "tassel",="" "gbs",="" "amplicon="" sequencing",="" "lepmap3",="" "genetic="" "genotype-by-sequencing",="" "rhampseq",="" "gwas",="" "biryani",="" "momo"="">

  <!-- favicon -->
  <link rel="shortcut icon" type="image/jpg" href="/image/avi.jpg">

  <!-- MathJax -->
  <script async="" src="https://www.google-analytics.com/analytics.js"></script><script type="text/x-mathjax-config;executed=true">MathJax.Hub.Config({messageStyle: 'none',tex2jax: {inlineMath: [['$','$'], ['\\(','\\)']]}});</script>
  <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.4/MathJax.js?config=TeX-MML-AM_CHTML" async=""></script>
  <script type="text/x-mathjax-config;executed=true">MathJax.Hub.Config({TeX: { equationNumbers: { autoNumber: "AMS" } }});</script>

  <link rel="alternate" type="application/rss+xml" title="Home - Personal website" href="/feed.xml">

  
  
<!-- Google Analytics -->
<script>
    (function (i, s, o, g, r, a, m) {
        i['GoogleAnalyticsObject'] = r; i[r] = i[r] || function () {
            (i[r].q = i[r].q || []).push(arguments)
        }, i[r].l = 1 * new Date(); a = s.createElement(o),
            m = s.getElementsByTagName(o)[0]; a.async = 1; a.src = g; m.parentNode.insertBefore(a, m)
    })(window, document, 'script', 'https://www.google-analytics.com/analytics.js', 'ga');
    ga('create', 'UA-123359651-1', 'auto');
    ga('send', 'pageview');
</script>
<!-- End Google Analytics -->

  
  
    
      
  <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">

    
  

  
    
      <link rel="stylesheet" href="/css/bootstrap.min.css">
    
      <link rel="stylesheet" href="/css/fonts.css">
    
      <link rel="stylesheet" href="/css/bootstrap-social.css">
    
      <link rel="stylesheet" href="/css/main.css">
    
      <link rel="stylesheet" href="/css/particle-js-style.css">
    
      <link rel="stylesheet" href="/fontawesome470/css/font-awesome.min.css">
    
  

  
    
      <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Lora:400,700,400italic,700italic">
    
      <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,600italic,700italic,800italic,400,300,600,700,800">
    
  

  

  

  

    <!-- Facebook OpenGraph tags -->
  

  
  <meta property="og:title" content="Avi Karn, Ph.D.">
  

   
  <meta property="og:description" content="Quantitative Geneticist &amp; Molecular plant breeding Postdoc @Cornell | Dr. Marcus Zuber endowment PhD fellow @Mizzou &amp; BSc @TrumanState">
  


  <meta property="og:type" content="website">

  
  <meta property="og:url" content="https://avinashkarn.github.io/avi.github.io/">
  <link rel="canonical" href="https://avinashkarn.github.io/avi.github.io/">
  

  
  <meta property="og:image" content="https://avinashkarn.github.io/avi.github.iokarn.jpg">
  
  

  <!-- Twitter summary cards -->
  <meta name="twitter:card" content="summary">
  <meta name="twitter:site" content="@avi_karn">
  <meta name="twitter:creator" content="@avi_karn">

  
  <meta name="twitter:title" content="Avi Karn, Ph.D.">
  

  
  <meta name="twitter:description" content="Quantitative Geneticist &amp; Molecular plant breeding Postdoc @Cornell | Dr. Marcus Zuber endowment PhD fellow @Mizzou &amp; BSc @TrumanState">
  

  
  <meta name="twitter:image" content="https://avinashkarn.github.io/avi.github.iokarn.jpg">
  

  
  
<script>try {
                        Object.defineProperty(screen, "availTop", { value: 0 });
                    } catch (e) {}
                    try {
                        Object.defineProperty(screen, "availLeft", { value: 0 });
                    } catch (e) {}
                    try {
                        Object.defineProperty(screen, "availWidth", { value: 1920 });
                    } catch (e) {}
                    try {
                        Object.defineProperty(screen, "availHeight", { value: 1080 });
                    } catch (e) {}
                    try {
                        Object.defineProperty(screen, "colorDepth", { value: 24 });
                    } catch (e) {}
                    try {
                        Object.defineProperty(screen, "pixelDepth", { value: 24 });
                    } catch (e) {}
                    try {
                        Object.defineProperty(navigator, "keyboard", { value: undefined });
                    } catch (e) {}
                    try {
                        Object.defineProperty(navigator, "hardwareConcurrency", { value: 8 });
                    } catch (e) {}
                    try {
                        Object.defineProperty(navigator, "deviceMemory", { value: 8 });
                    } catch (e) {}
                    try {
                        Object.defineProperty(navigator, "webkitTemporaryStorage", { value: undefined });
                    } catch (e) {}
                    try {
                        Object.defineProperty(navigator, "webkitPersistentStorage", { value: undefined });
                    } catch (e) {}
                    try {
                        Object.defineProperty(navigator, "appVersion", { value: "5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Brave Chrome/84.0.4147.89 Safari/537.36" });
                    } catch (e) {}
                    try {
                        Object.defineProperty(navigator, "doNotTrack", { value: null });
                    } catch (e) {}
                    
                navigator.getBattery = function getBattery () {
                let battery = {"charging":true,"chargingTime":0,"dischargingTime":null,"level":1}
            
                try {
                    Object.defineProperty(battery, 'onchargingchange', {
                        enumerable: true,
                        configurable: false,
                        writable: false,
                        value: undefined
                    })
                } catch (e) {}
                
                try {
                    Object.defineProperty(battery, 'onchargingtimechange', {
                        enumerable: true,
                        configurable: false,
                        writable: false,
                        value: undefined
                    })
                } catch (e) {}
                
                try {
                    Object.defineProperty(battery, 'ondischargingtimechange', {
                        enumerable: true,
                        configurable: false,
                        writable: false,
                        value: undefined
                    })
                } catch (e) {}
                
                try {
                    Object.defineProperty(battery, 'onlevelchange', {
                        enumerable: true,
                        configurable: false,
                        writable: false,
                        value: undefined
                    })
                } catch (e) {}
                
                    battery.addEventListener = function addEventListener () {
                        return
                    }
                
                return Promise.resolve(battery)
                }
            
            try {
                window.screenY = 864
            } catch (e) { }
        
            try {
                window.screenTop = 864
            } catch (e) { }
        
            try {
                window.top.window.outerHeight = window.screen.height
            } catch (e) { }
        
            try {
                window.screenX = 0
            } catch (e) { }
        
            try {
                window.screenLeft = 0
            } catch (e) { }
        
            try {
                window.top.window.outerWidth = window.screen.width
            } catch (e) { }
        </script><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 2px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 2px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: 1em}
.MathJax_MenuRadioCheck.RTL {right: 1em; left: auto}
.MathJax_MenuLabel {padding: 2px 2em 4px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #CCCCCC; margin: 4px 1px 0px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: Highlight; color: HighlightText}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Preview .MJXf-math {color: inherit!important}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1em; bottom: 1.5em; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom > * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd > * {display: block!important}
.MJXp-script-box > *  {display: table!important; height: 50%}
.MJXp-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXp-script-box > *:last-child > * {vertical-align: bottom}
.MJXp-script-box > * > * > * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over > * {display: block!important}
.MJXp-munderover > * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
.MathJax_PHTML .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style><script>try {
                        Object.defineProperty(screen, "availTop", { value: 0 });
                    } catch (e) {}
                    try {
                        Object.defineProperty(screen, "availLeft", { value: 0 });
                    } catch (e) {}
                    try {
                        Object.defineProperty(screen, "availWidth", { value: 1920 });
                    } catch (e) {}
                    try {
                        Object.defineProperty(screen, "availHeight", { value: 1080 });
                    } catch (e) {}
                    try {
                        Object.defineProperty(screen, "colorDepth", { value: 24 });
                    } catch (e) {}
                    try {
                        Object.defineProperty(screen, "pixelDepth", { value: 24 });
                    } catch (e) {}
                    try {
                        Object.defineProperty(navigator, "keyboard", { value: undefined });
                    } catch (e) {}
                    try {
                        Object.defineProperty(navigator, "hardwareConcurrency", { value: 8 });
                    } catch (e) {}
                    try {
                        Object.defineProperty(navigator, "deviceMemory", { value: 8 });
                    } catch (e) {}
                    try {
                        Object.defineProperty(navigator, "webkitTemporaryStorage", { value: undefined });
                    } catch (e) {}
                    try {
                        Object.defineProperty(navigator, "webkitPersistentStorage", { value: undefined });
                    } catch (e) {}
                    try {
                        Object.defineProperty(navigator, "appVersion", { value: "5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Brave Chrome/84.0.4147.89 Safari/537.36" });
                    } catch (e) {}
                    try {
                        Object.defineProperty(navigator, "doNotTrack", { value: null });
                    } catch (e) {}
                    
                navigator.getBattery = function getBattery () {
                let battery = {"charging":true,"chargingTime":0,"dischargingTime":null,"level":1}
            
                try {
                    Object.defineProperty(battery, 'onchargingchange', {
                        enumerable: true,
                        configurable: false,
                        writable: false,
                        value: undefined
                    })
                } catch (e) {}
                
                try {
                    Object.defineProperty(battery, 'onchargingtimechange', {
                        enumerable: true,
                        configurable: false,
                        writable: false,
                        value: undefined
                    })
                } catch (e) {}
                
                try {
                    Object.defineProperty(battery, 'ondischargingtimechange', {
                        enumerable: true,
                        configurable: false,
                        writable: false,
                        value: undefined
                    })
                } catch (e) {}
                
                try {
                    Object.defineProperty(battery, 'onlevelchange', {
                        enumerable: true,
                        configurable: false,
                        writable: false,
                        value: undefined
                    })
                } catch (e) {}
                
                    battery.addEventListener = function addEventListener () {
                        return
                    }
                
                return Promise.resolve(battery)
                }
            
            try {
                window.screenY = 864
            } catch (e) { }
        
            try {
                window.screenTop = 864
            } catch (e) { }
        
            try {
                window.top.window.outerHeight = window.screen.height
            } catch (e) { }
        
            try {
                window.screenX = 0
            } catch (e) { }
        
            try {
                window.screenLeft = 0
            } catch (e) { }
        
            try {
                window.top.window.outerWidth = window.screen.width
            } catch (e) { }
        </script></head>


<body><div id="MathJax_Message" style="display: none;"></div>
  <div class="mobile-js-hide">
    <!-- particles.js container -->
    <div id="particles-js"><canvas class="particles-js-canvas-el" width="1348" height="9351" style="width: 100%; height: 100%;"></canvas></div>
  </div>

    
    
    <nav class="navbar navbar-default navbar-fixed-top navbar-custom">
  <div class="container-fluid">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#main-navbar">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      
        <a class="navbar-brand" href="https://avinashkarn.github.io/avi.github.io">Home</a>
      
    </div>

    <div class="collapse navbar-collapse" id="main-navbar">
      <ul class="nav navbar-nav navbar-right">
      
        
          <li>
            






<a href="/resume">About Me / Contact</a>
          </li>
        
        
        
          <li class="navlinks-container" style="min-width: 113px;">
            <a class="navlinks-parent" href="javascript:void(0)">Tutorials</a>
            <div class="navlinks-children">
              
                
                  






<a href="https://avikarn.com/2020-06-26-MachineLearning_rf_glmnet/">Machine Learning 1</a>
                
              
                
                  






<a href="https://avikarn.com/2020-07-02-RNAseq_DeSeq2/">RNA-seq DF</a>
                
              
                
                  






<a href="https://avikarn.com/2019-04-17-Genetic-Mapping-in-Lep-MAP3/">Genetic Mapping</a>
                
              
                
                  






<a href="https://avikarn.com/2019-04-21-AmpSeq/">AmpSeq Concept</a>
                
              
                
                  






<a href="https://avikarn.com/2019-04-20-GBS-SNP-calling-tutorial/">GBS SNP calling</a>
                
              
                
                  






<a href="https://avikarn.com/2019-04-23-composite-interval-mapping/">QTL in R</a>
                
              
                
                  






<a href="https://avikarn.com/2019-08-22-ssr/">SSR in TASSEL</a>
                
              
                
                  






<a href="https://avikarn.com/2019-08-04-Mendel/">QC Mendelian errors</a>
                
              
                
                  






<a href="https://avikarn.com/2019-07-30-prunning/">Prune Markers</a>
                
              
                
                  






<a href="https://avikarn.com/2019-07-22-GWAS/">GWAS in TASSEL</a>
                
              
                
                  






<a href="https://avikarn.com/2019-07-09-ANOVA/">ANOVA in R</a>
                
              
                
                  






<a href="https://avikarn.com/2019-05-19-mapchart/">Plot Genetic Map</a>
                
              
                
                  






<a href="https://avikarn.com/2019-05-08-structure/">STRUCTURE software</a>
                
              
                
                  






<a href="https://avikarn.com/2019-05-06-MDS/">MDS analysis</a>
                
              
            </div>
          </li>
        
        
        
          <li class="navlinks-container" style="min-width: 135px;">
            <a class="navlinks-parent" href="javascript:void(0)">Publications</a>
            <div class="navlinks-children">
              
                
                  






<a href="https://scholar.google.com/citations?user=2XuofkoAAAAJ&amp;hl=en">Google Scholar Profile</a>
                
              
                
                  






<a href="https://www.researchgate.net/profile/Avinash_Karn">ResearchGate Profile</a>
                
              
                
                  






<a href="https://orcid.org/0000-0001-9730-8922">ORCID profile</a>
                
              
                
                  






<a href="https://publons.com/researcher/1700487/avinash-karn/">Publons Profile</a>
                
              
            </div>
          </li>
        
        
        
          <li class="navlinks-container" style="min-width: 134px;">
            <a class="navlinks-parent" href="javascript:void(0)">Hobbies</a>
            <div class="navlinks-children">
              
                
                  






<a href="/foodblog">Cooking</a>
                
              
                
                  






<a href="/astronomy">Astronomy</a>
                
              
                
                  






<a href="/photography">Photography</a>
                
              
                
                  






<a href="https://photos.app.goo.gl/TR82ChGjvV4rp3F16">Stamp Collection</a>
                
              
            </div>
          </li>
        
        
                  
      </ul>
    </div>

<!-- ALways use my avatar, irrespective of the post image -->
  <div class="avatar-container" style="display: block;">
    <div class="avatar-img-border">
      <a href="https://avinashkarn.github.io/avi.github.io">
        <img class="avatar-img" src="https://avinashkarn.github.io/avi.github.io/image/karn.jpg">
     </a>
    </div>
  </div>

  </div>
</nav>


    <!-- TODO this file has become a mess, refactor it -->




  <div id="header-big-imgs" data-num-img="11" data-img-src-1="/image/frontpage/rhampseq_core_genome.jpg" data-img-desc-1="Vitis core genome and rhAmpSeq genetic markers (Nature Comm 2020)" data-img-src-2="/image/frontpage/palm.jpg" data-img-desc-2="Collecting data" data-img-src-3="/image/frontpage/geneticMap.jpg" data-img-desc-3="Genetic Mapping" data-img-src-4="/image/frontpage/mlm.jpg" data-img-desc-4="Genome-wide association study" data-img-src-5="/image/frontpage/haploview.png" data-img-desc-5="Haploblock Haploview " data-img-src-6="/image/frontpage/sandiego.jpg" data-img-desc-6="Cliff Sunset beach, San Diego, CA" data-img-src-7="/image/frontpage/epistasis2.png" data-img-desc-7="Epsistatic interactions" data-img-src-8="/image/frontpage/grapes.png" data-img-desc-8="USDA-ARS Geneva Germplasm Repository 2018 Field day" data-img-src-9="/image/frontpage/circos_teonilNAM2.png" data-img-desc-9="Shared QTLs in TeoNILs and NAM for seed composition (Karn et al 2017)" data-img-src-10="/image/frontpage/20170622_113203.jpg" data-img-desc-10="Minirhizotron Tubes Installation" data-img-src-11="/image/frontpage/maizefield.jpg" data-img-desc-11="teosinte x maize trial"></div>


<header class="header-section has-img">

<div class="big-img intro-header" style="background-image: url(&quot;/image/frontpage/palm.jpg&quot;);">
  <div class="container">
    <div class="row">
      <div class="col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1">
        <div class="page-heading">
          <h1>Avi Karn, Ph.D.</h1>
		  
		    
            <hr class="small">
            <span class="page-subheading">Quantitative Geneticist &amp; Molecular plant breeding Postdoc @Cornell | Dr. Marcus Zuber endowment PhD fellow @Mizzou &amp; BSc @TrumanState</span>
			
		  
		  
		  
        </div>
      </div>
    </div>
  </div>
  <span class="img-desc" style="display: inline;">Collecting data</span>
</div>

<div class="intro-header no-img">
  <div class="container">
    <div class="row">
      <div class="col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1">
        <div class="page-heading">
          <h1>Avi Karn, Ph.D.</h1>
		  
		    
            <hr class="small">
            <span class="page-subheading">Quantitative Geneticist &amp; Molecular plant breeding Postdoc @Cornell | Dr. Marcus Zuber endowment PhD fellow @Mizzou &amp; BSc @TrumanState</span>
			
		  
		  
		  
        </div>
      </div>
    </div>
  </div>
</div>
</header>





<div class="container" role="main">
  <div class="row">
    <div class="col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1">
      <!-- site body (untouched)--> 
<div class="posts-list">
  
  <article class="post-preview">
    <a href="/2020-08-30-ImputationAccuracy/">
    <h2 class="post-title">Evaluate Genotype Imputation Accuracy in TASSEL GUI</h2>

    
    </a>

    <p class="post-meta">
      Posted on August 30, 2020
    </p>

    <div class="post-entry-container">
      
      <div class="post-image">
        <a href="/2020-08-30-ImputationAccuracy/">
          <img src="/image/imputeAccuracy/plotImputeAccuracy.png">
        </a>
      </div>
      
      <div class="post-entry">
        Compare imputation accuracies of different imputation methods (Beagle 5.1, LinkImputeR, kNNi, FILLIN) by calculating the percentage of correctly imputed genotypes in TASSEL GUI software. Please find more information on TASSEL software and its documentaton at this  Link 


        
        
          <a href="/2020-08-30-ImputationAccuracy/" class="post-read-more">[Read&nbsp;More]</a>
        
      </div>
    </div>
   
  
    
    <div class="blog-tags">
      Tags:
      
      
      <a href="/tags#Imputation">Imputation</a>
      
      <a href="/tags#GUI">GUI</a>
      
      <a href="/tags#Genotype">Genotype</a>
      
      <a href="/tags#TASSEL">TASSEL</a>
      
      <a href="/tags#Bioinformatics">Bioinformatics</a>
      
      
    </div>
    

   </article>
  
  <article class="post-preview">
    <a href="/2020-08-01-GenomicSelection_ML_H2_G2F/">
    <h2 class="post-title">Machine Learning algorithms for Genomic Selection of Quantitative Traits with varying Heritabilities</h2>

    
    </a>

    <p class="post-meta">
      Posted on August 1, 2020
    </p>

    <div class="post-entry-container">
      
      <div class="post-image">
        <a href="/2020-08-01-GenomicSelection_ML_H2_G2F/">
          <img src="/image/g2f_GS/Capture20.JPG">
        </a>
      </div>
      
      <div class="post-entry">
        Multiple Quantitative traits were evaluated with varying heritabilties to study how the inheritance of a trait affect the performance of genomic selection and prediction models. Machine learning algorithms Random Forest and GLMnet: Lasso and Elastic-Net Regularized Generalized Linear Models were deployed to develop Genomic selection models using the publicly available...
        
        
          <a href="/2020-08-01-GenomicSelection_ML_H2_G2F/" class="post-read-more">[Read&nbsp;More]</a>
        
      </div>
    </div>
   
  
    
    <div class="blog-tags">
      Tags:
      
      
      <a href="/tags#Genomic Selection">Genomic Selection</a>
      
      <a href="/tags#Machine Learning">Machine Learning</a>
      
      <a href="/tags#Random Forest">Random Forest</a>
      
      <a href="/tags#GLMnet">GLMnet</a>
      
      <a href="/tags#Heritability">Heritability</a>
      
      <a href="/tags#Quantitative Trait">Quantitative Trait</a>
      
      
    </div>
    

   </article>
  
  <article class="post-preview">
    <a href="/2020-07-02-RNAseq_DeSeq2/">
    <h2 class="post-title">RNA-Seq data analysis in R - Investigate differentially expressed genes in your data! </h2>

    
    </a>

    <p class="post-meta">
      Posted on July 2, 2020
    </p>

    <div class="post-entry-container">
      
      <div class="post-image">
        <a href="/2020-07-02-RNAseq_DeSeq2/">
          <img src="/image/rna/Capture14.PNG">
        </a>
      </div>
      
      <div class="post-entry">
        In this tutorial, negative binomial was used to perform differential gene expression analyis in R using DESeq2, pheatmap and tidyverse packages. The workflow for the RNA-Seq data is: Obatin the FASTQ sequencing files from the sequencing facilty Assess the quality of the sequencing reads Perform genome alignment to identify the...
        
        
          <a href="/2020-07-02-RNAseq_DeSeq2/" class="post-read-more">[Read&nbsp;More]</a>
        
      </div>
    </div>
   
  
    
    <div class="blog-tags">
      Tags:
      
      
      <a href="/tags#RNA-Seq">RNA-Seq</a>
      
      <a href="/tags#Differential Expression">Differential Expression</a>
      
      <a href="/tags#DESeq">DESeq</a>
      
      <a href="/tags#Data analysis">Data analysis</a>
      
      
    </div>
    

   </article>
  
  <article class="post-preview">
    <a href="/2020-06-26-MachineLearning_rf_glmnet/">
    <h2 class="post-title">Utilizing Machine Learning algorithms (GLMnet and Random Forest models) for Genomic Prediction of a Quantitative trait</h2>

    
    </a>

    <p class="post-meta">
      Posted on June 26, 2020
    </p>

    <div class="post-entry-container">
      
      <div class="post-image">
        <a href="/2020-06-26-MachineLearning_rf_glmnet/">
          <img src="/image/mlr_rf/plot11.JPG">
        </a>
      </div>
      
      <div class="post-entry">
        In this tutorial, I used two popular machine learning algorithms: Random Forest and GLMnet for Genomic Prediction of a quantitative trait. There are several machine learning R packages available, however, in this tutorial i used caret package. The objective was to develop two models: Random forest and glmnet using real...
        
        
          <a href="/2020-06-26-MachineLearning_rf_glmnet/" class="post-read-more">[Read&nbsp;More]</a>
        
      </div>
    </div>
   
  
    
    <div class="blog-tags">
      Tags:
      
      
      <a href="/tags#Machine Learning">Machine Learning</a>
      
      <a href="/tags#GLMNET">GLMNET</a>
      
      <a href="/tags#LASSO">LASSO</a>
      
      <a href="/tags#Random Forest">Random Forest</a>
      
      <a href="/tags#Genomic Prediction">Genomic Prediction</a>
      
      <a href="/tags#Quantitative traits">Quantitative traits</a>
      
      
    </div>
    

   </article>
  
  <article class="post-preview">
    <a href="/2020-05-26-correlation_shiny/">
    <h2 class="post-title">Correlation analysis using R Shiny Application</h2>

    
    </a>

    <p class="post-meta">
      Posted on May 26, 2020
    </p>

    <div class="post-entry-container">
      
      <div class="post-image">
        <a href="/2020-05-26-correlation_shiny/">
          <img src="/image/shiny/plot_corr.png">
        </a>
      </div>
      
      <div class="post-entry">
        R Shiny statistical output visualization app, where a user can upload their own numeric data in .csv or .txt formats, and generate a correlation matrix plots that also contains density plots and printed coefficient of correlation values. Please check it out by uploading your own data below.


        
        
          <a href="/2020-05-26-correlation_shiny/" class="post-read-more">[Read&nbsp;More]</a>
        
      </div>
    </div>
   
  
    
    <div class="blog-tags">
      Tags:
      
      
      <a href="/tags#R">R</a>
      
      <a href="/tags#Shiny">Shiny</a>
      
      <a href="/tags#Shinyapps">Shinyapps</a>
      
      <a href="/tags#Data products">Data products</a>
      
      <a href="/tags#Correlation">Correlation</a>
      
      
    </div>
    

   </article>
  
  <article class="post-preview">
    <a href="/2020-01-21-rhAmpSeq_natureComm/">
    <h2 class="post-title">Publication: Haplotyping the Vitis collinear core genome with rhAmpSeq improves marker transferability in a diverse genus</h2>

    
    </a>

    <p class="post-meta">
      Posted on January 21, 2020
    </p>

    <div class="post-entry-container">
      
      <div class="post-image">
        <a href="/2020-01-21-rhAmpSeq_natureComm/">
          <img src="/image/rhAmpSeq/circos.png">
        </a>
      </div>
      
      <div class="post-entry">
        Abstract: Transferable DNA markers are essential for breeding and genetics. Grapevine (Vitis) breeders utilize disease resistance alleles from congeneric species ~20 million years divergent, but existing Vitis marker platforms have cross-species transfer rates as low as 2%. Here, we apply a marker strategy targeting the inferred Vitis core genome. Incorporating...
        
        
          <a href="/2020-01-21-rhAmpSeq_natureComm/" class="post-read-more">[Read&nbsp;More]</a>
        
      </div>
    </div>
   
  
    
    <div class="blog-tags">
      Tags:
      
      
      <a href="/tags#rhAmpSeq">rhAmpSeq</a>
      
      <a href="/tags#Vitis">Vitis</a>
      
      <a href="/tags#Universal Marker">Universal Marker</a>
      
      <a href="/tags#Genetic diversiy">Genetic diversiy</a>
      
      <a href="/tags#Amplicon sequencing">Amplicon sequencing</a>
      
      
    </div>
    

   </article>
  
  <article class="post-preview">
    <a href="/2019-08-22-ssr/">
    <h2 class="post-title">Import SSR marker data in TASSEL software</h2>

    
    </a>

    <p class="post-meta">
      Posted on August 22, 2019
    </p>

    <div class="post-entry-container">
      
      <div class="post-image">
        <a href="/2019-08-22-ssr/">
          <img src="/image/ssr/ssr.png">
        </a>
      </div>
      
      <div class="post-entry">
        In this tutorial, I would like to share how one can import SSR (simple sequence repeats) marker data on to TASSEL software, and perfrom analyses such as imputation, General Linear Model and Principle Component Analysis.


        
        
          <a href="/2019-08-22-ssr/" class="post-read-more">[Read&nbsp;More]</a>
        
      </div>
    </div>
   
  
    
    <div class="blog-tags">
      Tags:
      
      
      <a href="/tags#SSR">SSR</a>
      
      <a href="/tags#TASSEL">TASSEL</a>
      
      <a href="/tags#GLM">GLM</a>
      
      <a href="/tags#PCA">PCA</a>
      
      
    </div>
    

   </article>
  
  <article class="post-preview">
    <a href="/2019-08-04-Mendel/">
    <h2 class="post-title">Check for mendelian errors using pedigree and genotype information </h2>

    
    </a>

    <p class="post-meta">
      Posted on August 4, 2019
    </p>

    <div class="post-entry-container">
      
      <div class="post-image">
        <a href="/2019-08-04-Mendel/">
          <img src="/image/bcftools/plot.png">
        </a>
      </div>
      
      <div class="post-entry">
        Cluster based methods such as multidimensional scaling (MDS) and priniciple component analysis (PCA) are traditionally used in identifying samples with genotypic inconsistencies, however, it is important to identify genotypes with high mendelian inconsistencies prior to any genetic or statistical analysis. In this tutorial, I would like to share how one...
        
        
          <a href="/2019-08-04-Mendel/" class="post-read-more">[Read&nbsp;More]</a>
        
      </div>
    </div>
   
  
    
    <div class="blog-tags">
      Tags:
      
      
      <a href="/tags#Genetic data">Genetic data</a>
      
      <a href="/tags#Mendelian Error">Mendelian Error</a>
      
      <a href="/tags#bcftools">bcftools</a>
      
      
    </div>
    

   </article>
  
  <article class="post-preview">
    <a href="/2019-07-30-prunning/">
    <h2 class="post-title">Pruning genetic markers based on their physical distance and linkage disequilibrium (LD)</h2>

    
    </a>

    <p class="post-meta">
      Posted on July 30, 2019
    </p>

    <div class="post-entry-container">
      
      <div class="post-image">
        <a href="/2019-07-30-prunning/">
          <img src="/image/prune/prune.png">
        </a>
      </div>
      
      <div class="post-entry">
        High density markers do not provide any additional information, therefore, can be pruned based on the physical distances between adjacent markers and linkage disequilibrium (LD). In this tutorial, I will show how to prune markers based on their physical position in TASSEL software, and based on LD in PLINK software....
        
        
          <a href="/2019-07-30-prunning/" class="post-read-more">[Read&nbsp;More]</a>
        
      </div>
    </div>
   
  
    
    <div class="blog-tags">
      Tags:
      
      
      <a href="/tags#Genetic Markers">Genetic Markers</a>
      
      <a href="/tags#Pruning">Pruning</a>
      
      <a href="/tags#PLINK">PLINK</a>
      
      <a href="/tags#TASSEL">TASSEL</a>
      
      <a href="/tags#GUI">GUI</a>
      
      <a href="/tags#Linkage disequilibrium">Linkage disequilibrium</a>
      
      
    </div>
    

   </article>
  
  <article class="post-preview">
    <a href="/2019-07-22-GWAS/">
    <h2 class="post-title">Genome-wide Association Study (GWAS) in TASSEL (GUI)</h2>

    
    </a>

    <p class="post-meta">
      Posted on July 22, 2019
    </p>

    <div class="post-entry-container">
      
      <div class="post-image">
        <a href="/2019-07-22-GWAS/">
          <img src="/image/gwas/gwaslogo.png">
        </a>
      </div>
      
      <div class="post-entry">
        TASSEL aslo known as Trait Analysis by aSSociation, Evolution and Linkage is a powerful statistical software to conduct association mapping such as General Linear Model (GLM) and Mixed Linear Model (MLM). The GUI (graphical user interface) version of TASSEL is very well built for anyone who does not have a...
        
        
          <a href="/2019-07-22-GWAS/" class="post-read-more">[Read&nbsp;More]</a>
        
      </div>
    </div>
   
  
    
    <div class="blog-tags">
      Tags:
      
      
      <a href="/tags#GWAS">GWAS</a>
      
      <a href="/tags#TASSEL">TASSEL</a>
      
      <a href="/tags#GUI">GUI</a>
      
      <a href="/tags#Genome-wide association test">Genome-wide association test</a>
      
      <a href="/tags#GLM">GLM</a>
      
      <a href="/tags#MLM">MLM</a>
      
      <a href="/tags#Linear model">Linear model</a>
      
      
    </div>
    

   </article>
  
  <article class="post-preview">
    <a href="/2019-07-09-ANOVA/">
    <h2 class="post-title">ANOVA and Tukey test in R software in just few steps!</h2>

    
    </a>

    <p class="post-meta">
      Posted on July 9, 2019
    </p>

    <div class="post-entry-container">
      
      <div class="post-image">
        <a href="/2019-07-09-ANOVA/">
          <img src="/image/anova/anova.png">
        </a>
      </div>
      
      <div class="post-entry">
        ANOVA also known as Analysis of Variance is a powerful statistical method to test a hypothesis involving more than two groups (also known as treatments). However, ANOVA is limited in providing a detailed insights between different treatments or groups, and this is where, Tukey (T) test also known as T-test...
        
        
          <a href="/2019-07-09-ANOVA/" class="post-read-more">[Read&nbsp;More]</a>
        
      </div>
    </div>
   
  
    
    <div class="blog-tags">
      Tags:
      
      
      <a href="/tags#ANOVA">ANOVA</a>
      
      <a href="/tags#T-test">T-test</a>
      
      <a href="/tags#Tukey Test">Tukey Test</a>
      
      <a href="/tags#statistics">statistics</a>
      
      <a href="/tags#data analysis">data analysis</a>
      
      
    </div>
    

   </article>
  
  <article class="post-preview">
    <a href="/2019-06-19-MAS_AmpSeq/">
    <h2 class="post-title">Marker Assisted Selection using AmpSeq data</h2>

    
    </a>

    <p class="post-meta">
      Posted on June 19, 2019
    </p>

    <div class="post-entry-container">
      
      <div class="post-image">
        <a href="/2019-06-19-MAS_AmpSeq/">
          <img src="/image/ampseq/Keep2.jpg">
        </a>
      </div>
      
      <div class="post-entry">
        This tutorial video is my attempt to introduce the basics of the AmpSeq data analyis in Marker Assisted Selection . 
For general information on technology behind AmpSeq please read one of my previous blog post on it at this link: https://avikarn.com/2019-04-21-AmpSeq/


        
        
          <a href="/2019-06-19-MAS_AmpSeq/" class="post-read-more">[Read&nbsp;More]</a>
        
      </div>
    </div>
   
  
    
    <div class="blog-tags">
      Tags:
      
      
      <a href="/tags#Marker Assisted Selection">Marker Assisted Selection</a>
      
      <a href="/tags#AmpSeq">AmpSeq</a>
      
      <a href="/tags#Amplicon Sequencing">Amplicon Sequencing</a>
      
      
    </div>
    

   </article>
  
  <article class="post-preview">
    <a href="/2019-05-19-mapchart/">
    <h2 class="post-title">Plot Genetic Maps in few steps using MapChart software</h2>

    
    </a>

    <p class="post-meta">
      Posted on May 19, 2019
    </p>

    <div class="post-entry-container">
      
      <div class="post-image">
        <a href="/2019-05-19-mapchart/">
          <img src="/image/mapchart/qtlplot.PNG">
        </a>
      </div>
      
      <div class="post-entry">
        MapChart is a free software to plot publishing quality genetic maps as well as QTL data. This software was developed by Roeland E. Voorrips at Wageningen University and can be downloaded at this  link .


        
        
          <a href="/2019-05-19-mapchart/" class="post-read-more">[Read&nbsp;More]</a>
        
      </div>
    </div>
   
  
    
    <div class="blog-tags">
      Tags:
      
      
      <a href="/tags#MapChart software">MapChart software</a>
      
      <a href="/tags#Genetic Map plot">Genetic Map plot</a>
      
      
    </div>
    

   </article>
  
  <article class="post-preview">
    <a href="/2019-05-08-structure/">
    <h2 class="post-title">Investigate genetic admixture using STRUCTURE software</h2>

    
    </a>

    <p class="post-meta">
      Posted on May 8, 2019
    </p>

    <div class="post-entry-container">
      
      <div class="post-image">
        <a href="/2019-05-08-structure/">
          <img src="/image/structure/finalstructure.jpg">
        </a>
      </div>
      
      <div class="post-entry">
        Structure Software is a freely available software package that one may use for rigorous investigation of admixed individuals; identification of point of hybridization and migrants; and estimate over all structure of a population using a commonly used genetic markers such as SNPs and SSRs. This software was developed by Pritchard...
        
        
          <a href="/2019-05-08-structure/" class="post-read-more">[Read&nbsp;More]</a>
        
      </div>
    </div>
   
  
    
    <div class="blog-tags">
      Tags:
      
      
      <a href="/tags#Structure software">Structure software</a>
      
      <a href="/tags#Genetic Admixture">Genetic Admixture</a>
      
      <a href="/tags#Genetic data">Genetic data</a>
      
      <a href="/tags#Population structure">Population structure</a>
      
      
    </div>
    

   </article>
  
  <article class="post-preview">
    <a href="/2019-05-06-MDS/">
    <h2 class="post-title">How to use Multidimensional Scaling (MDS) to quality control your genetic data?</h2>

    
    </a>

    <p class="post-meta">
      Posted on May 6, 2019
    </p>

    <div class="post-entry-container">
      
      <div class="post-image">
        <a href="/2019-05-06-MDS/">
          <img src="/image/mds/mdsanimation1.gif">
        </a>
      </div>
      
      <div class="post-entry">
        Multidimensional Scaling (MDS) is a powerful statistical method that can be effectively used to elucidate hidden population structure, and more importantly, use it as a quality control tool while working on genetic data.


        
        
          <a href="/2019-05-06-MDS/" class="post-read-more">[Read&nbsp;More]</a>
        
      </div>
    </div>
   
  
    
    <div class="blog-tags">
      Tags:
      
      
      <a href="/tags#Multidimensional Scaling">Multidimensional Scaling</a>
      
      <a href="/tags#MDS">MDS</a>
      
      <a href="/tags#Genetic data">Genetic data</a>
      
      <a href="/tags#QC">QC</a>
      
      
    </div>
    

   </article>
  
  <article class="post-preview">
    <a href="/2019-04-23-composite-interval-mapping/">
    <h2 class="post-title">QTL mapping using Composite Interval Mapping (CIM) method in R software</h2>

    
    </a>

    <p class="post-meta">
      Posted on April 23, 2019
    </p>

    <div class="post-entry-container">
      
      <div class="post-image">
        <a href="/2019-04-23-composite-interval-mapping/">
          <img src="/image/cim/scan.png">
        </a>
      </div>
      
      <div class="post-entry">
        In the Quantitative Trait Locus (QTL) analysis, composite interval mapping (CIM) method estimates the QTL positon with higher accuracy and statistical significance by combining interval mapping with multiple regression. This method also controls background noise resulting from genetic variations in other regions of the genome that affect the detection of...
        
        
          <a href="/2019-04-23-composite-interval-mapping/" class="post-read-more">[Read&nbsp;More]</a>
        
      </div>
    </div>
   
  
    
    <div class="blog-tags">
      Tags:
      
      
      <a href="/tags#Composite Interval Mapping">Composite Interval Mapping</a>
      
      <a href="/tags#QTL mapping">QTL mapping</a>
      
      <a href="/tags#R">R</a>
      
      <a href="/tags#r/qtl">r/qtl</a>
      
      
    </div>
    

   </article>
  
  <article class="post-preview">
    <a href="/2019-04-21-AmpSeq/">
    <h2 class="post-title">Amplicon Sequencing (AmpSeq): Concept and data analysis</h2>

    
    </a>

    <p class="post-meta">
      Posted on April 21, 2019
    </p>

    <div class="post-entry-container">
      
      <div class="post-image">
        <a href="/2019-04-21-AmpSeq/">
          <img src="/image/ampseq/ampseq_process.png">
        </a>
      </div>
      
      <div class="post-entry">
        Amplicon Sequencing (AmpSeq) technology combines highly multiplexed PCR sequences of multiple barcoded samples in a single reaction. Amplicons include SNP, haplotypes, SSRs and presence/absence variants. Currently, in a single reaction about 400 amplicons and 3000 samples can be processed simultaneously. In this blog, my goal is to basically introduce the...
        
        
          <a href="/2019-04-21-AmpSeq/" class="post-read-more">[Read&nbsp;More]</a>
        
      </div>
    </div>
   
  
    
    <div class="blog-tags">
      Tags:
      
      
      <a href="/tags#Amplicon sequencing">Amplicon sequencing</a>
      
      <a href="/tags#AmpSeq">AmpSeq</a>
      
      <a href="/tags#Multiplex PCR">Multiplex PCR</a>
      
      <a href="/tags#Genetic Marker">Genetic Marker</a>
      
      
    </div>
    

   </article>
  
  <article class="post-preview">
    <a href="/2019-04-20-GBS-SNP-calling-tutorial/">
    <h2 class="post-title">Call GBS SNPs in 7 steps using TASSEL GBSv2 pipeline</h2>

    
    </a>

    <p class="post-meta">
      Posted on April 20, 2019
    </p>

    <div class="post-entry-container">
      
      <div class="post-image">
        <a href="/2019-04-20-GBS-SNP-calling-tutorial/">
          <img src="/image/GBS/GBS.png">
        </a>
      </div>
      
      <div class="post-entry">
        Genotype-by-Sequencing (GBS) is reduced representation of a genome, which utilizes restriction enzymes (e.g. ApeKI) and NextGen sequencing to identify biallelic markers and presence/absence markers. In this post, my attempt is to consisely present the GBS SNP calling process in 7 steps using the TASSEL GBSv2 pipeline. Pleae note:, Buckler et...
        
        
          <a href="/2019-04-20-GBS-SNP-calling-tutorial/" class="post-read-more">[Read&nbsp;More]</a>
        
      </div>
    </div>
   
  
    
    <div class="blog-tags">
      Tags:
      
      
      <a href="/tags#GBS">GBS</a>
      
      <a href="/tags#SNP calling">SNP calling</a>
      
      <a href="/tags#TASSEL">TASSEL</a>
      
      <a href="/tags#Bioinformatics">Bioinformatics</a>
      
      
    </div>
    

   </article>
  
  <article class="post-preview">
    <a href="/2019-04-17-Genetic-Mapping-in-Lep-MAP3/">
    <h2 class="post-title">Genetic map construction in LepMap3</h2>

    
    </a>

    <p class="post-meta">
      Posted on April 17, 2019
    </p>

    <div class="post-entry-container">
      
      <div class="post-image">
        <a href="/2019-04-17-Genetic-Mapping-in-Lep-MAP3/">
          <img src="/image/lepmap/cropped_nih_geneticmap.jpg">
        </a>
      </div>
      
      <div class="post-entry">
        Building genetic maps can be challenging and sometimes quite stressful, especially, when dealing with thousands or even millions of markers. In this post, I am hoping to help anyone who would like to get started to build a decent genetic map in an open software Lep-MAP3 , and finally, evaluating...
        
        
          <a href="/2019-04-17-Genetic-Mapping-in-Lep-MAP3/" class="post-read-more">[Read&nbsp;More]</a>
        
      </div>
    </div>
   
  
    
    <div class="blog-tags">
      Tags:
      
      
      <a href="/tags#Lep-MAP3">Lep-MAP3</a>
      
      <a href="/tags#Genetic Mapping">Genetic Mapping</a>
      
      <a href="/tags#Mapchart">Mapchart</a>
      
      
    </div>
    

   </article>
  
  <article class="post-preview">
    <a href="/2019-04-02-winemag/">
    <h2 class="post-title">In the news: How Computers are Being Used to Hack Grapes</h2>

    
    </a>

    <p class="post-meta">
      Posted on April 2, 2019
    </p>

    <div class="post-entry-container">
      
      <div class="post-image">
        <a href="/2019-04-02-winemag/">
          <img src="/image/winemag.JPG">
        </a>
      </div>
      
      <div class="post-entry">
        I was interviewed by the Wine Enthusiast Magazine on their special edition on âWine and Techâ, where i had an opportunity to represent our  VitisGen2 project  and talk about how computers are being used to âhackâ Grapes!


        
        
          <a href="/2019-04-02-winemag/" class="post-read-more">[Read&nbsp;More]</a>
        
      </div>
    </div>
   
  
    
    <div class="blog-tags">
      Tags:
      
      
      <a href="/tags#Wine Ethusiast Magzine">Wine Ethusiast Magzine</a>
      
      <a href="/tags#Winemag">Winemag</a>
      
      <a href="/tags#VitisGen">VitisGen</a>
      
      <a href="/tags#Avi Karn">Avi Karn</a>
      
      <a href="/tags#Computers">Computers</a>
      
      
    </div>
    

   </article>
  



<ul class="pager main-pager">
  
  
  <li class="next">
    <a href="/page2">Older Posts â</a>
  </li>
  
</ul>


<div style="max-width:95%">
  <p><div style="max-width: 341px; margin: 0px auto;"><div style="position: relative; padding-top: 100%;"><iframe style="background: transparent !important; position: absolute; left: 0px; top: 0px;" scrolling="no" frameborder="0" allowtransparency="true" width="100%" height="100%" src="//ra.revolvermaps.com/w/6/a/a2.php?i=0rn8vq73z9x&amp;m=7&amp;c=ff0000&amp;cr1=ffffff&amp;f=arial&amp;l=1"></iframe></div></div><script type="text/javascript" src="//ra.revolvermaps.com/0/0/6.js?i=0rn8vq73z9x&amp;m=7&amp;c=ff0000&amp;cr1=ffffff&amp;f=arial&amp;l=1" async="async"></script></p>
</div>
 
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async="" src="https://www.googletagmanager.com/gtag/js?id=UA-123359651-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());
  gtag('config', 'UA-123359651-1');
</script>

<script async="" src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
<script>
  (adsbygoogle = window.adsbygoogle || []).push({
    google_ad_client: "ca-pub-5126027065024936",
    enable_page_level_ads: true
  });
</script>

	    
    </div>
  </div>
</div>





    <!-- Begin Mailchimp Signup Form -->
<link href="//cdn-images.mailchimp.com/embedcode/horizontal-slim-10_7.css" rel="stylesheet" type="text/css">
<style type="text/css">
	#mc_embed_signup{background:#fff; clear:left; font:14px Helvetica,Arial,sans-serif; width:100%;}
	/* Add your own Mailchimp form style overrides in your site stylesheet or in this style block.
	   We recommend moving this block and the preceding CSS link to the HEAD of your HTML file. */
</style>

<!--End mc_embed_signup-->
<footer>
	
  <div class="container beautiful-jekyll-footer">
    <div class="row">
      <div class="col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1">
        <ul class="list-inline text-center footer-links">
          
          <li>
            <a href="mailto:avi.karn6@gmail.com" title="Email me">
              <span class="fa-stack fa-lg" aria-hidden="true">
                <i class="fa fa-circle fa-stack-2x"></i>
                <i class="fa fa-envelope fa-stack-1x fa-inverse"></i>
              </span>
              <span class="sr-only">Email me</span>
            </a>
          </li>
          
          
          <li>
            <a href="https://linkedin.com/in/avikarn" title="LinkedIn">
              <span class="fa-stack fa-lg" aria-hidden="true">
                <i class="fa fa-circle fa-stack-2x"></i>
                <i class="fa fa-linkedin fa-stack-1x fa-inverse"></i>
              </span>
              <span class="sr-only">LinkedIn</span>
            </a>
          </li>
          
          
      
          <li>
            <a href="https://twitter.com/avi_karn" title="Twitter">
              <span class="fa-stack fa-lg" aria-hidden="true">
                <i class="fa fa-circle fa-stack-2x"></i>
                <i class="fa fa-twitter fa-stack-1x fa-inverse"></i>
              </span>
              <span class="sr-only">Twitter</span>
            </a>
          </li>
          
    
      

      
      
      
      
      
          
          <li>
            <a href="https://www.facebook.com/avi.karn" title="Facebook">
              <span class="fa-stack fa-lg" aria-hidden="true">
                <i class="fa fa-circle fa-stack-2x"></i>
                <i class="fa fa-facebook fa-stack-1x fa-inverse"></i>
              </span>
              <span class="sr-only">Facebook</span>
            </a>
          </li>
          
      
      
      
      
      
      
        </ul>
        
      <p class="copyright text-muted">
        
      Â© Avinash Karn
      &nbsp;â¢&nbsp;
      2020
      
      &nbsp;â¢&nbsp;
      <a href="https://avinashkarn.github.io/avi.github.io">https://AviKarn.com</a>
      
      






<a href="/"></a>
      </p>
	<div class="footer__copyright">
		<p style="text-align:center">
		<a href="/Terms_Conditions">Terms &amp; Conditions</a>
		<a href="/privacy_policy">Privacy Policy</a>
		</p>
		
	</div>
	
<!-- Please don't remove this, keep my open source work credited :) -->
      <p class="theme-by text-muted">
      Awesome Theme by
      <a href="https://deanattali.com/beautiful-jekyll/">beautiful-jekyll</a>
      </p>
      </div>
    </div>
  </div>
</footer>

  
    






  
    <!-- doing something a bit funky here because I want to be careful not to include JQuery twice! -->
    
      <script>
        if (typeof jQuery == 'undefined') {
          document.write('<script src="/js/jquery-1.12.4.min.js"></scr' + 'ipt>');
        }
      </script><script src="/js/jquery-1.12.4.min.js"></script>
    
  
    <!-- doing something a bit funky here because I want to be careful not to include JQuery twice! -->
    
  <script src="/js/bootstrap.min.js"></script>
    
  
    <!-- doing something a bit funky here because I want to be careful not to include JQuery twice! -->
    
  <script src="/js/main.js"></script>
    
  
    <!-- doing something a bit funky here because I want to be careful not to include JQuery twice! -->
    
  <script src="/js/particles.min.js"></script>
    
  
    <!-- doing something a bit funky here because I want to be careful not to include JQuery twice! -->
    
  <script src="/js/particles-config.js"></script>
    
  

  

</div></body></html>